src/core/payment-integration/payment-integration-service.ts

```diff
@@ -0,0 +1,21 @@
+import { BillingAddressRequestBody } from '../../billing';
+import { OrderRequestBody } from '../../order';
+import { ShippingAddressRequestBody } from '../../shipping';
+
+import PaymentIntegrationSelectors from './payment-integration-selectors';
+
+export default interface PaymentIntegrationService {
```

Comment:
What about methods related to payments, eg submitPayment, getPaymentMethod? Do we expose the `_paymentMethodActionCreator` and `_paymentActionCreator` to the strategy packages?

Reply:
Thanks, good spot, I missed them.